config/initializers/assets.rb

```diff
@@ -9,8 +9,3 @@
 # Add additional assets to the asset load path
 # Rails.application.config.assets.paths << Emoji.images_path
 
-# Precompile additional assets (application.js.coffee.erb, application.css, and all non-JS/CSS are already added)
-Rails.application.config.assets.precompile += %w( diagram.js graphing.js map_marker.js ace.js )
```

Comment:
I see that you added that. But how does it know to generate `diagram.js`, `graphing.js`, `map_marker.js`, `ace.js`, etc. as final output packages so that they can be served? We link them directly, e.g. https://github.com/cantino/huginn/blob/5868c7b4b23516bce7f7a9867b2310d6ef85980b/app/views/agents/agent_views/user_location_agent/_show.html.erb#L3